### ISSUE TITLE
background is getting repeated if core-icon-button src is pointing to an external SVG

### ISSUE THREAD

**Issue by saan1984** [Polymer/polymer#784]
<core -icon-button src="<url to svg>">
</core -icon-button>
and can be solved by adding the below css in the core-icon-button module.
:host(#resource) core-icon div{
    background-repeat: no-repeat;
}

**Comment by atdotslashdot** [comment]
I'm not sure where the best place is to hit this fix but i added
    icon.style.backgroundRepeat = 'no-repeat';

to the srcChanged function in core-icon.html
```
srcChanged: function() {
      var icon = this._icon || document.createElement('div');
      icon.textContent = '';
      icon.setAttribute('fit', '');
      icon.style.backgroundImage = 'url(' + this.src + ')';
      icon.style.backgroundPosition = 'center';
      icon.style.backgroundSize = '100%';
      icon.style.backgroundRepeat = 'no-repeat';
      if (!icon.parentNode) {
        this.appendChild(icon);
      }
      this._icon = icon;
    },
```

**Comment by Sinjins** [comment]
I have implemented the same fix as atdotslashdot.  (i.e. adding icon.style.backgroundRepeat = 'no-repeat' to core-icon's 'srcChanged' function), and it seems to do the trick.  Is this the recommended approach?  If so, can we expect that to be implemented in the master branch soon?  I'm using 0.5.4 right now...haven't checked it if is in 0.5.5 yet so my apologies if it is.

**Comment by tjsavage** [comment]
Closing this issue due to age and the release of [version 1](https://www.polymer-project.org/1.0/) of Polymer - please feel free to re-open if this is incorrect.
